### DISCUSSION TITLE
Welcome to 2022-Better-Working-World-Data-Challenge Discussions!

### DISCUSSION THREAD

**Discussion by tecklowdna** [EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rkDevguru47** [comment]
On the website, level 1 submission file 'challenge_1_test_regions.txt' is missing or I am having trouble finding it. The only file we get from [](url)https://challenge.ey.com/api/v1/storage/admin-files/7814487399971564-621ca093a8b0820013279231-Level_1_challenge_1_submission_template.csv is challenge_1_submission_template.csv 
pls help

**Reply by rkDevguru47** [reply]
> On the website, level 1 submission file 'challenge_1_test_regions.txt' is missing or I am having trouble finding it. The only file we get from https://challenge.ey.com/api/v1/storage/admin-files/7814487399971564-621ca093a8b0820013279231-Level_1_challenge_1_submission_template.csv is challenge_1_submission_template.csv pls help

got it, it was in the repo!!

**Comment by ibtihelgharsalah** [comment]
Heyy guys,
I'm Ibtihel from Tunisia, I am a junior student at Tunis Business School and I am to start with this level1 challenge .. I wish yall luck and a very enriching experience !

**Comment by jonasanmaru** [comment]
Hi all,

Anyone had any issues playing the challenge on local environment?

I use the following the guide from:
https://github.com/EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge/wiki/1c.-Getting-started:-Local-Environment

```
conda create -n 2022DSC anaconda 
conda create -n 2022ey python=3.8.12 anaconda 
conda install contextily xarray zarr aiohttp requests pystac pystac-client planetary-computer nb_conda_kernels scikit-learn fsspec shapely GDAL=3.2 rioxarray stackstac xarray-spatial
```

Tested with both Python 3.8.12 and Python 3.9,
I keep getting package conflict when installing the additional package when running conda install contextily xarray .... 

![imcompatible-package2](https://user-images.githubusercontent.com/16115274/164746838-22c5de26-2ce7-4808-8c5b-7fc1ab939e77.PNG)
![incompatibel-package](https://user-images.githubusercontent.com/16115274/164746846-97fb5f8e-8e26-4c42-9cdb-0ac5983b6aa5.PNG)

**Reply by BillyDodds** [reply]
Hi @jonasanmaru . Sorry to hear about your issue. Can I confirm that you ran `conda activate 2022ey` before the `conda install ...` step?

**Reply by jonasanmaru** [reply]
Hi @BillyDodds, yes, I have run conda activate 2022ey (other env name), tested with both python 3.8.12 and python 3.9.
I will try it out with pip install later on as suggested by the other participants, and see how it goes. 
![image](https://user-images.githubusercontent.com/16115274/166120133-d00650bf-860f-47f8-a6b1-6239c4635a8f.png)

**Reply by jonasanmaru** [reply]
@BillyDodds  yes, I just tried again, I still received incompatible packages message.
Python 3.8.12
![image](https://user-images.githubusercontent.com/16115274/166139199-552ca2d2-2d5a-46e4-b602-26b1fc0370e6.png)

**Reply by BillyDodds** [reply]
Hi @jonasanmaru . 

The dependencies in this challenge have been a nightmare, so I apologise for what I imagine has been a very frustrating experience. I can assure you I went through the same frustrating process to arrive at the requirements.txt for the challenge. From my testing, there appears to be a very sensitive dependence with xarray, rioxarray, and rasterio that is also sensitive to the python version. It might be worth trying to `pip install` these packages or others to bypass the conda environment solve. Theres a possibility things might break down the line, but it might be better than nothing. 

The following is the requirements.txt file created on the Azure VM environment that reflects the recommended setup detailed in the challenge wiki. This definitely works for the Azure VM environment, and might be worth inspecting when troubleshooting your local installation.
 [requirements.txt](https://github.com/EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge/files/8608390/requirements.txt)

I'm also curious about what happens when you let the examination process go. It might take ages, but does it eventually finish?

Billy

**Reply by jonasanmaru** [reply]
Hi @BillyDodds,

No worries, I tried using pip install, but  some packages seems to be having issues as well.
Anyway, I have let it run for at least two days, it never finished.

![image](https://user-images.githubusercontent.com/16115274/166422082-ba043507-164a-4b0a-babd-04e4263b61c7.png)

I guess, I will just proceed with Azure or colab to complete the tasks.
Thank you for your advice and time, I will check on the Azure VM environment and see if it works on local machine.

**Reply by BillyDodds** [reply]
Hi @jonasanmaru . 

I've added a different requirements file to the repo. Please see [this discussion post](https://github.com/EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge/discussions/5#discussion-4050755) for details. Please let me know if those instructions fix your issues.

Billy

**Comment by ibtihelgharsalah** [comment]
Yes me too

**Comment by ibtihelgharsalah** [comment]
Am I the only one who lost access to the challenge page on the EY data challenge website? are you guys facing the same issue?

**Reply by jonasanmaru** [reply]
Same here!

**Reply by BillyDodds** [reply]
Hi @ibtihelgharsalah and @jonasanmaru. Is the challenge page still unavailable to you both? If so, could you provide some more details on which page in particular or how we can replicate the issue? Thanks!
